### DISCUSSION TITLE
Compare PINTS with the software in BCM (may also be useful source of MCMC algorithms)

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1539] [Ideas/Science]
The attached paper which discusses 'BCM' software makes it look quite similar to what we are proposing in PINTS. I haven't been through it in detail, and it doesn't look that popular, so may not be anything to worry about. Either way, however, it may be a useful source of MCMC [algorithms](https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-016-0339-3) for PINTS.

**Comment by ben18785** [comment]
A few things make me unsure about the quality of this paper. For example,

"Time required for generating 1000 samples from the posterior using BCM, SYSBIONS, BioBayes, ABC-SysBio and Stan, with several different sampling algorithm. The sampling was terminated if it had not converged after 7 days."

This is a terrible metric. It is unsurprising that their SMC methods beat Stan by this criterion. I can generate 1000 samples from any given model in less than a second. The question as to the quality of those samples is what matters. How many 'effective samples' per second are generated is what matters. (Effective samples are the number of independent samples generated per second.)

**Comment by ben18785** [comment]
Another package uses parallel tempering, and is called [BioBayes](http://www.dcs.gla.ac.uk/BioBayes/).

One called [SysBions](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4325544/pdf/btu675.pdf) from Michael Stumpf's group uses nested sampling.

**Comment by MichaelClerx** [comment]
Is this still on the table?
